src/uu/env/src/native_int_str.rs

```diff
@@ -0,0 +1,325 @@
+// This file is part of the uutils coreutils package.
+//
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+
+// This module contains classes and functions for dealing with the differences
```

Comment:
this can't be used  elsewhere in the coreutils ?

Reply:
I was planning to move it as soon as I found some other usecase for it.
But If you wish, I move it also in advance.